ArduPlane/servos.cpp

```diff
@@ -547,9 +547,7 @@ void Plane::set_servos_controlled(void)
                control_mode == &mode_fbwa ||
                control_mode == &mode_autotune) {
         // a manual throttle mode
-        if (!rc().has_valid_input()) {
```

Comment:
Now were headlining it in the `get_throttle_input` and `get_adjusted_throttle_input` calls we don't need this check.